pkg/config/config.go

```diff
@@ -64,19 +64,32 @@ type Config struct {
 
 // TODO: https://github.com/jbrekelmans/kube-compose/issues/64
 // nolint
-func New() (*Config, error) {
-	fileName := "docker-compose.yml"
-	data, err := ioutil.ReadFile(fileName)
-	if err != nil {
+func New(file *string) (*Config, error) {
+	var data []byte
+	var err error
+	var fileName string
+	if file != nil {
+		if *file == "" {
+			return nil, fmt.Errorf("docker-compose file flag cannot be empty string")
+		}
+		data, err = ioutil.ReadFile(*file)
```

Comment:
If `err` is not a `IsNotExist` error then it is ignored.

Reply:
Good catch, that's not corrected.